docs/Changelog.md

```diff
@@ -8024,6 +7996,92 @@ This version of the operator has been available since version 7 of the default O
 </dl>
 
 ## Version 8 of the default ONNX operator set
+### <a name="ConvInteger-8"></a>**ConvInteger-8**</a>
+
+  The integer convolution operator consumes an input tensor, a filter and a padding value, and computes the output. The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.
```

Comment:
`The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.`
An ONNX backend can't guarantee that result never overflows (which is implied by "MUST"). IMO, a better wording would be "If the any intermediate computation overflows 32-bit integer, the result is implementation-defined"

Reply:
Give output data type is 32 bits, we're assuming that production between 2 8-bits should never overflow. Make sense? if there're exceptional cases, the behavior not specified will be undefined by default.

Reply:
Ok, I see, it would be better to phrase the first sentence as `The multiplication never overflows`. As for the second part, lets not introduce any undefined behavior, it is a horrible C++ concept that even C++ standardization committee tries to eradicate. It would suggest "If the any intermediate computation overflows 32-bit integer, the result is implementation-defined": this hints that operation must still produce some result (and not e.g. crash the whole system, which would be OK with undefined behavior), but this results depends on the backend implementation and is generally not portable.